cartography/intel/gcp/iam.py

```diff
@@ -186,6 +186,17 @@ def get_roles(iam: Resource, project_id: str) -> List[Dict]:
             page = res.get('roles', [])
             roles.extend(page)
             req = iam.roles().list_next(previous_request=req, previous_response=res)
+
+        req = crm_v1.projects().get(projectId=project_id)
+        res_project = req.execute()
+        if res_project.get('parent').get('type') == 'organization':
```

Comment:
we should fetch roles at folder level as well. right?

Reply:
for folder level role there is no special api we can get this by `iam.roles().list(view="FULL")` method